platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java

```diff
@@ -36,5 +36,5 @@
 @ConfigData("consensus")
 public record ConsensusConfig(
         @ConfigProperty(defaultValue = "26") int roundsNonAncient,
-        @ConfigProperty(defaultValue = "2000") int roundsExpired,
+        @ConfigProperty(defaultValue = "500") int roundsExpired,
```

Comment:
I think this should be 5000 to match 0.50

Reply:
500 i think you mean

Reply:
Old value is 500